spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+
+RSpec.describe VehicleFactory do
+  it "exists" do
+    factory = VehicleFactory.new
+    expect(factory).to be_an_instance_of(VehicleFactory)
+  end
+
+  it "creates a vehicle from a list of one vehicle" do
```

Comment:
You were really smart about how you tested this. Start small. We just need to test from one small piece of data that the `create_vehicles` method works. Then you incremented in the next test to have multiple vehicles before finally testing the external data set. Really nice work.